integration_test/dapp_tests/dapp_tests.sh

```diff
@@ -20,5 +20,6 @@ npx hardhat compile
 # Set the CONFIG environment variable
 export DAPP_TEST_ENV=$1
 
-npx hardhat test --network $1 uniswap/uniswapTest.js
-npx hardhat test --network $1 steak/SteakTests.js
+#npx hardhat test --network $1 uniswap/uniswapTest.js
```

Comment:
is this supposed to be commented out? also maybe we can add another argument to specify which test to run - and if no argument is passed, it just runs all 3?

Reply:
Oops, forgot to add it back in when I was testing something - just updated to take in the argument so it's easier to run locally